doc/source/image_description/elements.rst

```diff
@@ -582,14 +592,9 @@ overlayroot="true|false"
   resized on first boot to the available disk space.
 
 overlayroot_verity_blocks="number|all"
-  For the `oem` type only, specifies to create a dm verity hash
-  from the number of given blocks (or all) placed at the end of the squashfs
-  compressed read-only root filesystem. For later verification of the device,
-  the credentials information produced by `veritysetup` from the
-  cryptsetup tools are needed. This data as of now is only printed
-  as debugging information to the build log file. A concept to
-  persistently store the verification metadata as part of the
-  partition(s) will be a next step.
+  Same as the `verity_blocks` attribute but the verity hash is
```

Comment:
What happens if `overlayroot_verity_blocks` and `verity_blocks` are both used at the same time? I am asking because I wonder if we could get rid of `overlyroot_verity_blocks` in favor of only having `verity_blocks` and just behave as it used to do in case of overlayroot.

I am not verify familiar with this feature hence I am not sure I miss something.

Reply:
done